apps/api/v2/src/modules/organizations/teams/verified-resources/org-teams-verified-resources.controller.ts

```diff
@@ -0,0 +1,242 @@
+import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
+import { Roles } from "@/modules/auth/decorators/roles/roles.decorator";
+import { ApiAuthGuard } from "@/modules/auth/guards/api-auth/api-auth.guard";
+import { PlatformPlanGuard } from "@/modules/auth/guards/billing/platform-plan.guard";
+import { IsAdminAPIEnabledGuard } from "@/modules/auth/guards/organizations/is-admin-api-enabled.guard";
+import { IsOrgGuard } from "@/modules/auth/guards/organizations/is-org.guard";
+import { RolesGuard } from "@/modules/auth/guards/roles/roles.guard";
+import { IsTeamInOrg } from "@/modules/auth/guards/teams/is-team-in-org.guard";
+import { RequestEmailVerificationInput } from "@/modules/verified-resources/inputs/request-email-verification.input";
+import { RequestPhoneVerificationInput } from "@/modules/verified-resources/inputs/request-phone-verification.input";
+import { VerifyEmailInput } from "@/modules/verified-resources/inputs/verify-email.input";
+import { VerifyPhoneInput } from "@/modules/verified-resources/inputs/verify-phone.input";
+import { RequestEmailVerificationOutput } from "@/modules/verified-resources/outputs/request-email-verification-output";
+import { RequestPhoneVerificationOutput } from "@/modules/verified-resources/outputs/request-phone-verification-output";
+import {
+  VerifiedEmailOutput,
+  VerifiedEmailOutputData,
+  VerifiedEmailsOutput,
+} from "@/modules/verified-resources/outputs/verified-email.output";
+import {
+  VerifiedPhoneOutput,
+  VerifiedPhoneOutputData,
+  VerifiedPhonesOutput,
+} from "@/modules/verified-resources/outputs/verified-phone.output";
+import { VerifiedResourcesService } from "@/modules/verified-resources/services/verified-resources.service";
+import {
+  Body,
+  Controller,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  ParseIntPipe,
+  Post,
+  Query,
+  UseGuards,
+} from "@nestjs/common";
+import { ApiHeader, ApiOperation, ApiTags } from "@nestjs/swagger";
+import { plainToClass } from "class-transformer";
+
+import { ERROR_STATUS, SUCCESS_STATUS } from "@calcom/platform-constants";
+import { SkipTakePagination } from "@calcom/platform-types";
+
+@Controller({
+  path: "/v2/organizations/:orgId/teams/:teamId/verified-resources",
+})
+@UseGuards(ApiAuthGuard, IsOrgGuard, RolesGuard, IsTeamInOrg, PlatformPlanGuard, IsAdminAPIEnabledGuard)
+@ApiTags("Organization Team Verified Resources")
+export class OrgTeamsVerifiedResourcesController {
+  constructor(private readonly verifiedResourcesService: VerifiedResourcesService) {}
+  @ApiOperation({
+    summary: "Request Email Verification Code",
+    description: `Sends a verification code to the email.`,
+  })
+  @Roles("TEAM_ADMIN")
+  @Throttle({
+    limit: 3,
+    ttl: 60000,
+    blockDuration: 60000,
+    name: "org_teams_verified_resources_emails_requests",
+  })
+  @PlatformPlan("ESSENTIALS")
+  @ApiHeader(API_KEY_OR_ACCESS_TOKEN_HEADER)
+  @Post("/emails/request")
```

Comment:
Word "request" is bit vague in my opinion. How about we call this endpoint `/emails/verification-code` because we are not requesting an email but creating a verificaiton code that is sent to the user.

And then rename `@Post("/emails/verify")` below to `@Post("/emails/verification-code/verify")`. to better show that they are meant to be used 1 after another and are linked.